hack/run-tests.sh

```diff
@@ -9,3 +9,4 @@ tag=v0.7.0
 kubeconfig=~/.kube/config
 
 ${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -test.timeout 60m
+${TESTS_OUT_DIR}/cdi.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -test.timeout 60m
```

Comment:
Add a newline to the end of the file please.

Reply:
done